standard/tier/tier_utils.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Utils
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- module intended to be moved to `Module:Tier` after the old ones usage has been eliminated
+
+local HiddenSort = require('Module:HiddenSort')
+local Logic = require('Module:Logic')
+local Page = require('Module:Page')
+local String = require('Module:StringUtils')
+
+local TierData = mw.loadData('Module:Tier/Data')
+
+local NON_BREAKING_SPACE = '&nbsp;'
+
+local Tier = {}
+
+--- Converts input to standardized identifier format
+---@param input string|integer|nil
+---@return string|integer|nil
+function Tier.toIdentifier(input)
+	if String.isEmpty(input) then
+		return ''
+	end
+
+	if Logic.isNumeric(input) then
+		return tonumber(input)
+	end
+
+	return string.lower(input):gsub(' ', '')
+end
+
+--- Retrieves the raw tier/tierType data for a given input
+---@param input string|integer|nil
+---@param mode 'tiers'|'tiertypes'
+---@return table?
+function Tier.raw(input, mode)
```

Comment:
```suggestion
function Tier.getData(tier)
```

or

```suggestion
function Tier.data(tier)
```

Reply:
that would mean we would need to duplicate it for tiertype too ...
not a fan